Source/Rhetos.Deployment/DataMigrationScriptsFromDisk.cs

```diff
@@ -81,7 +104,10 @@ public List<DataMigrationScript> Load()
                     "Data migration scripts '{0}' and '{1}' have same tag '{2}' in their headers.",
                     badGroup.First().Path, badGroup.ElementAt(1).Path, badGroup.Key));
 
-            return allScripts;
+            string serializedMigrationScripts = JsonConvert.SerializeObject(allScripts, Formatting.Indented);

```

Comment:
Please add performance logger for this code block. "DataMigrationScriptsFromDisk: Saved {_scripts.Count} scripts to generated file.", or something similar.